### PR TITLE
Added functionality to run Config Validator by running docker command

### PR DESCRIPTION
**Pull request by red2k18**
Resolves #3690 and part of #3248 

Updated `initialize_forseti_services.sh` script to run a docker command to run CV instead of static binary.

### CODE REVIEW

File: modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl [hunk 1]
```diff
@@ -39,7 +39,12 @@ FORSETI_COMMAND+=" --forseti_db $SQL_SERVER_LOCAL_ADDRESS/${cloudsql_db_name}?ch
 FORSETI_COMMAND+=" --config_file_path ${forseti_server_conf_path}"
 FORSETI_COMMAND+=" --services $FORSETI_SERVICES"
 
-CONFIG_VALIDATOR_COMMAND="${forseti_home}/external-dependencies/config-validator/ConfigValidatorRPCServer"
+CONFIG_VALIDATOR_COMMAND="$(which docker) run -d"
+CONFIG_VALIDATOR_COMMAND+=" --log-driver=gcplogs"
+CONFIG_VALIDATOR_COMMAND+=" --log-opt gcp-log-cmd=true"
+CONFIG_VALIDATOR_COMMAND+=" --log-opt labels=git-sync"
```
Comment (gkowalski-google) [inline modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl]: The log labels for CV should be different than for git-sync, can we change the label to config-validator?

Comment (red2k18) [reply modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl]: Done

**Comment by red2k18** [comment]
image

**Comment by red2k18** [comment]
Verified that the `config-validator` service is running from another instance:
image

File: modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl [hunk 2]
```diff
@@ -39,10 +39,15 @@ FORSETI_COMMAND+=" --forseti_db $SQL_SERVER_LOCAL_ADDRESS/${cloudsql_db_name}?ch
 FORSETI_COMMAND+=" --config_file_path ${forseti_server_conf_path}"
 FORSETI_COMMAND+=" --services $FORSETI_SERVICES"
 
-CONFIG_VALIDATOR_COMMAND="${forseti_home}/external-dependencies/config-validator/ConfigValidatorRPCServer"
+CONFIG_VALIDATOR_COMMAND="$(which docker) run -p 50052:50052 --name config-validator"
```
Comment (kevensen) [inline modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl]: Add a `--rm` between `run` and `-p 50052:50052`

File: modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl [hunk 3]
```diff
@@ -98,7 +100,7 @@ CONFIG_VALIDATOR_SERVICE="$(cat << EOF
 [Unit]
 Description=Config Validator API Server
 [Service]
-User=ubuntu
+TimeoutStartSec=10s
```
Comment (kevensen) [inline modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl]: Increase this to 15s

File: modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl [hunk 4]
```diff
@@ -98,7 +100,7 @@ CONFIG_VALIDATOR_SERVICE="$(cat << EOF
...
 Environment="GOGC=1000"
 ExecStart=$CONFIG_VALIDATOR_COMMAND
```
Comment (kevensen) [inline modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl]: Add `ExecStop=/usr/bin/docker kill config-validator`

File: modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl (hunk 2 above)
Comment (red2k18) [reply modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl]: Done

File: modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl (hunk 4 above)
Comment (red2k18) [reply modules/server/templates/scripts/forseti-server/initialize_forseti_services.sh.tpl]: Done

**Comment by red2k18** [comment]
Config Validator scanner is running successfully.
image

Config Validator scanner is generating violations.
image

**Comment by red2k18** [comment]
Integration tests for Config Validator Scanner also passed.
image

**Pull request merged**